src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -1,142 +1,144 @@
-    assertEquals(-0.8999999761581421, termRatingEntity.getProperty("score-professor"));
-  }
-
-  @Test
-  public void addTermRating_overwritingExistingTermRating() throws IOException {
-
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    request = Mockito.mock(HttpServletRequest.class);
-    Entity termEntity = new Entity("Term");
-    datastore.put(termEntity);
-    Key termKey = termEntity.getKey();
-
-    when(request.getParameter("term")).thenReturn(KeyFactory.keyToString(termKey));
-    when(request.getParameter("term-input")).thenReturn("I really like this class.");
-    when(request.getParameter("rating-term")).thenReturn("4");
-    when(request.getParameter("hoursOfWork")).thenReturn("8");
-    when(request.getParameter("difficulty")).thenReturn("4");
-    when(request.getParameter("prof-input")).thenReturn("The professor was okay.");
-    when(request.getParameter("rating-professor")).thenReturn("3");
-    when(request.getParameter("ID")).thenReturn("numberOneId");
-
-    AnalyzeSentimentResponse response =
-        AnalyzeSentimentResponse.newBuilder()
-            .setDocumentSentiment(Sentiment.newBuilder().setScore((float) -0.699999988079071))
-            .build();
-    when(languageService.analyzeSentiment(any(Document.class))).thenReturn(response);
-
-    newTermRating.addTermRating(request);
-
-    when(request.getParameter("term-input")).thenReturn("I don't like this class.");
-    when(request.getParameter("rating-term")).thenReturn("1");
-    when(request.getParameter("hoursOfWork")).thenReturn("10");
-    when(request.getParameter("difficulty")).thenReturn("5");
-    when(request.getParameter("prof-input")).thenReturn("This teacher was wonderful.");
-    when(request.getParameter("rating-professor")).thenReturn("3");
-    when(request.getParameter("ID")).thenReturn("numberOneId");
-    when(request.getParameter("term")).thenReturn(KeyFactory.keyToString(termKey));
-
-    newTermRating.addTermRating(request);
-
-    Entity termRatingEntity =
-        newTermRating.queryEntities("Rating", "reviewer-id", "numberOneId").get(0);
-    assertEquals("I don't like this class.", termRatingEntity.getProperty("comments-term"));
-    assertEquals("numberOneId", termRatingEntity.getProperty("reviewer-id"));
-    assertEquals(Long.valueOf(1), termRatingEntity.getProperty("perception-term"));
-    assertEquals(Long.valueOf(10), termRatingEntity.getProperty("hours"));
-    assertEquals(Long.valueOf(5), termRatingEntity.getProperty("difficulty"));
-    assertEquals("This teacher was wonderful.", termRatingEntity.getProperty("comments-professor"));
-    assertEquals(Long.valueOf(3), termRatingEntity.getProperty("perception-professor"));
-    assertEquals(-0.699999988079071, termRatingEntity.getProperty("score-term"));
-    assertEquals(-0.699999988079071, termRatingEntity.getProperty("score-professor"));
-  }
-}
+// package com.google.sps.servlets;
```

Comment:
Please either delete or leave it alone. Don't commit commented code. Version control has a copy of it if you need it later.

Reply:
File is supposed to be there but didn't want to run test cases, will un-comment now!